Tests/Perf/metrics-test.yml

```diff
@@ -11,4 +11,4 @@ startupTimeTest:
 
 binarySizeTest:
   diffMin: 200 KiB
-  diffMax: 420 KiB
+  diffMax: 600 KiB
```

Comment:
Because the test is loading a dynamic framework now, its initial size is bigger than the previous solution with SPM.